app/assets/stylesheets/govuk_publishing_components/components/_layout-super-navigation-header.scss

```
@@ -542,38 +542,37 @@ $chevron-indent-spacing: 7px;
 
 // Styles for top level navigation toggle button.
 .gem-c-layout-super-navigation-header__navigation-top-toggle-button {
-  @include govuk-font($size: 19, $weight: "bold", $line-height: 20px);
+  @include govuk-link-common;
+  @include govuk-link-style-no-visited-state;
+  @include govuk-link-style-no-underline;
+
+  font-size: 16px;
+  font-size: govuk-px-to-rem(16px);
+  font-weight: 700;
   background: govuk-colour("black");
   border: 0;
-  border-left: 1px solid govuk-colour("white");
-  border-right: 1px solid govuk-colour("white");
   box-sizing: border-box;
   color: govuk-colour("white");
   cursor: pointer;
   height: govuk-spacing(9);
-  margin-right: -1px;
-  padding: govuk-spacing(4);
+  padding: 0;
   position: absolute;
-  right: (govuk-spacing(9) - govuk-spacing(3));
+  right: ((govuk-spacing(9) - govuk-spacing(3)) - 1px);
```

I think that this is 44px - do you think it's worth adding that as a comment:
```suggestion
  right: ((govuk-spacing(9) - govuk-spacing(3)) - 1px); // 44px
```

or just using 44px with an explanation? (Realise I should've done the explanation in a previous commit).

```suggestion
  right: 44px; // width of the search button 60px - (border of 1px + margin of 15px)
```